cli.js

```diff
@@ -0,0 +1,14 @@
+const { Command } = require('commander');
+const fs = require('fs');
+const exportModule = require('./exportModule.js');
+
+const program = new Command();
+program.version('0.0.1');
+function data() {
+  program.option('--elmJSON <file>', 'elm json to turn to synthea module').parse(process.argv);
+  const contents = JSON.parse(fs.readFileSync(program.elmJSON, 'utf8'));
+  return contents;
```

Comment:
Hmm maybe we can surround these inside a try/catch? Just incase there is an issue with parsing the json. If we get into the catch block, we can just log that there was a problem parsing the json and then call `process.exit(1)` which will stop execution of everything